src/pages/Admin/admin.module.scss

```diff
@@ -8,7 +8,7 @@
 }
 
 .content {
-  width: 100%;
+  width: calc(100% - 60px);
```

Comment:
Are you sure it won't break anything?

Reply:
As it affects only on admin page - yes, I am sure